modules/calc/src/main/java/com/opengamma/strata/calc/config/ReportingCurrency.java

```diff
@@ -23,63 +24,85 @@
 import org.joda.beans.impl.direct.DirectMetaProperty;
 import org.joda.beans.impl.direct.DirectMetaPropertyMap;
 
-import com.opengamma.strata.basics.CalculationTarget;
 import com.opengamma.strata.basics.currency.Currency;
+import com.opengamma.strata.calc.runner.function.CalculationFunction;
 
 /**
- * Reporting rules that always returns the same currency.
+ * The reporting currency.
  * <p>
- * These rules always return the same currency from {@link #reportingCurrency}.
+ * This is used to specify the currency that the result should be reporting in.
+ * The currency specified may be explicit, using {@link #of(Currency)}, or implicit
+ * using {@link #NATURAL}.
```

Comment:
Probably worth mentioning where the natural currency comes from here as well as in the comments for `NATURAL`.